tmt/steps/discover/shell.py

```diff
@@ -115,21 +115,37 @@ def from_raw(cls: Type[T], raw_data: Dict[str, Any], logger: tmt.utils.Common) -
 
         return data
 
-    def to_serialized(self) -> Dict[str, Any]:
-        """
-        Return keys and values in the form allowing later reconstruction.
+    def to_raw(self) -> Dict[str, Any]:
```

Comment:
In #1522 I've noticed that the `to_raw()` methods were renamed to `to_spec()`. Shall we use the new naming here?

Reply:
I'd leave that to https://github.com/teemtee/tmt/pull/1522 which is based on this patch anyway. So yeah, the name added here is not perfect, but #1522 will fix that.